templates/default/less/variables.less

```diff
@@ -443,6 +443,7 @@
 @ilFormColor: #808080;
 @ilFormBgColor: #FDFDFD;
 @input-color-placeholder: lighten(@gray-light, 30%);
+@il-textarea-height: (floor(4 * (@il-standard-form-font-size * @line-height-base)) + (@padding-base-vertical * 2) + 2);
```

Comment:
Took me some time to understand it, but makes sense, thanks for introducing a variable for this!